api/server/server.go

```diff
@@ -214,11 +214,21 @@ func postContainersPause(eng *engine.Engine, version version.Version, w http.Res
 	if err := parseForm(r); err != nil {
 		return err
 	}
-	job := eng.Job("pause", vars["name"])
-	if err := job.Run(); err != nil {
+
+	name := vars["name"]
```

Comment:
no need to be checked...handler for this route is only run if name is not empty...same below for unpause...